cirq/devices/grid_qubit_test.py

```diff
@@ -32,6 +32,36 @@ def test_xmon_qubit_eq():
     eq.make_equality_group(lambda: cirq.GridQubit(50, 25))
 
 
+def test_square():
+    assert cirq.GridQubit.square(2, top=1) == [cirq.GridQubit(1, 0),
+                                               cirq.GridQubit(1, 1),
+                                               cirq.GridQubit(2, 0),
+                                               cirq.GridQubit(2, 1)]
+
+
+def test_rec():
+    assert cirq.GridQubit.rect(1, 2, top=1, left=2) == [cirq.GridQubit(1, 2),
```

Comment:
Avoid repeating argument values in tests, or use multiple calls. Also check default behavior.

Reply:
Adjusted argument values, added test for default behavior.